service/src/main/java/tds/config/configuration/ConfigServiceApplicationConfiguration.java

```diff
@@ -0,0 +1,15 @@
+package tds.config.configuration;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+import tds.common.configuration.RestTemplateConfiguration;
+import tds.common.web.advice.ExceptionAdvice;
+
+/**
+ * This is the base configuration class for the config microservice
+ */
+@Configuration
+@Import({ExceptionAdvice.class, RestTemplateConfiguration.class})
+public class ConfigServiceApplicationConfiguration {
```

Comment:
My thinking in this class is that rather than clutter up our Application runner class that we should instead have a base configuration.  This would be where we would configure any specific beans that we don't get with auto wiring.